opentracing-api/src/main/java/io/opentracing/propagation/Format.java

```diff
@@ -70,6 +76,14 @@
          * @see io.opentracing.Tracer#extract(Format, Object)
          * @see Format
          */
-        public final static Format<ByteBuffer> BINARY = new Builtin<ByteBuffer>();
+        public final static Format<ByteBuffer> BINARY = new Builtin<ByteBuffer>("BINARY");
+
+        /**
+         * @return Short name for built-in formats as they tend to show up in exception messages.
+         */
+        @Override
+        public String toString() {
+            return "Builtin." + name;
```

Comment:
@bensigelman Thanks for the mention; I'll modify the PR; this change is trivial, looks like I missed @pavolloffay 's comment..